hole/fixed-tests/flags.txt

```diff
@@ -0,0 +1,747 @@
+< Afghanistan
```

Comment:
Rather than another giant copy of the data I wonder if we should create a go file that just looks up the DataMap at runtime and transforms it into the Run struct.